packages/expo-facebook/android/src/main/java/expo/modules/facebook/FacebookModule.java

```diff
@@ -146,14 +197,14 @@ public void onCancel() {
       public void onError(FacebookException error) {
         LoginManager.getInstance().registerCallback(mCallbackManager, null);
 
-        promise.reject(error);
+        promise.reject("ERR_FACEBOOK_LOGIN", "An error occurred while trying to log in to Facebook", error);
```

Comment:
I like extracting tags to static constants — it makes them easier to maintain — and I see you like them too (eg. it works like this on iOS). Would you care to do so here too?